front/cypress/e2e/project_page_ideas.cy.ts

```diff
@@ -212,7 +212,7 @@ describe('Archived timeline project with ideation phase', () => {
           title: projectTitle,
           descriptionPreview: projectDescriptionPreview,
           description: randomString(),
-          publicationStatus: 'archived',
+          publicationStatus: 'draft',
```

Comment:
Creating it as  draft to allow creating an idea and then changing it to archive such that we test showing archived ideas